src/main/java/Pool.java

```diff
@@ -0,0 +1,19 @@
+import vi.viredis.client.ViRedis;
+
+public interface Pool<T> {
+
+    /*
+     * @return one of the pooled objects.
+     */
+    ViRedis get();
+
+    /*
+     * @param object T to be return back to pool
+     */
+    void release(T object);
```

Comment:
Don't use the term 'release' when you mean 'return'. Futhermore, what is the difference between 'return' and 'put'?

Reply:
done!